cpp/CMakeLists.txt

```diff
@@ -84,7 +84,6 @@ add_compile_options(-Wall)
 add_compile_options(-Wno-sign-compare)
 add_compile_options(-Wno-comment)
 
-add_compile_options(-Werror)
```

Comment:
why remove this option?

Reply:
There was some error throwed while compiling with gcc12.1